### PR TITLE
Add handy option to turn on direct I/O in db_bench

### PR DESCRIPTION
**Pull request by omegaga**
These are two handy options I used in my experiment. Sending this PR in hope that others might find it helpful when they also want to use direct I/O (for bypassing OS page cache or other purposes).

### CODE REVIEW

**Comment by yiwu-arbug** [comment]
You need to actually set these options in `InitializeOptionsFromFlags()`.

**Comment by omegaga** [comment]
Seems that I didn't push the correct version of my code. Fixing it.

**Comment by omegaga** [comment]
Updated. Also fixed mmap_read and mmap_write initialization in db_bench.

**Comment by siying** [comment]
CC @IslamAbdelRahman 

The directIO read should be working. RocksDB is not ready for directIO write yet.

**Comment by omegaga** [comment]
@siying That's exactly what we observed last week. Do you think it better to remove the use_direct_writes, or leave it as a future option?

**Comment by IslamAbdelRahman** [comment]
@omegaga, Yes let's remove `use_direct_writes` to not confuse users, and add it later when we support it

**Comment by IslamAbdelRahman** [comment]
@omegaga, can you please look at the test failures ?

**Comment by omegaga** [comment]
Fixed. Seems that current failed test is unrelated.

**Comment by yiwu-arbug** [comment]
LGTM. Will merge after sandcastle job finishes.

@IslamAbdelRahman do we have land bot yet, or shall I stick to github merge?

File: util/cf_options.h [hunk 1]
```diff
@@ -69,6 +69,9 @@ struct ImmutableCFOptions {
   // Allow the OS to mmap file for writing. Default: false
   bool allow_mmap_writes;
 
+  // Use O_DIRECT for reading file
+  bool use_direct_reads;
```
Comment (yiwu-arbug) [inline util/cf_options.h]: BTW seems you don't need the option in CF options. Remove this line?

**Pull request merged**